tools/external/wabt/src/tools/postpass.cc

```diff
@@ -90,20 +90,92 @@ uint32_t GetStackPtr( Module& mod, const std::vector<uint8_t>& buff ) {
    return stack_ptr;
 }
 
-void StripZeroedData( Module& mod, size_t& fix_bytes ) {
-   std::vector<DataSegment*> ds;
-   for ( auto DS : mod.data_segments ) {
-      bool isZeroed = true;
-      for ( auto datum : DS->data ) {
-         isZeroed &= datum == 0;
-      }
-      if (!isZeroed) {
-         ds.push_back(DS);
+inline bool IsZeroed(const DataSegment* ds) {
+   bool is_zero = true;
+   for ( auto d : ds->data ) {
+      is_zero &= d == 0;
```

Comment:
Yep, that one was because it started life as a different function that was keeping some more information at a per byte case in the loop and forgot to update this.